wsd/Storage.cpp

```diff
@@ -908,6 +908,13 @@ WopiStorage::WOPIFileInfo::WOPIFileInfo(const FileInfo& fileInfo,
     (void)uriObject;
 #endif
 
+    // disable file renaming in readonly mode.
+    if (!_userCanWrite)
+    {
+        _userCanNotWriteRelative = true;
+        _userCanRename = false;
```

Comment:
is that setting per user or per file?
(it's important, if we have 2 users and one of them has edit permission, we should still allow that)
(maybe we should check user permission when command to rename / save as comes to the server?)